azure_functions_worker/dispatcher.py

```diff
@@ -567,25 +567,28 @@ def tp_max_workers_validator(value: str) -> bool:
                                'integer')
                 return False
 
-            if int_value < PYTHON_THREADPOOL_THREAD_COUNT_MIN or (
-                    int_value > PYTHON_THREADPOOL_THREAD_COUNT_MAX):
+            if int_value < PYTHON_THREADPOOL_THREAD_COUNT_MIN:
                 logger.warning(f'{PYTHON_THREADPOOL_THREAD_COUNT} must be set '
                                f'to a value between '
                                f'{PYTHON_THREADPOOL_THREAD_COUNT_MIN} and '
-                               f'{PYTHON_THREADPOOL_THREAD_COUNT_MAX}. '
-                               'Reverting to default value for max_workers')
+                               'sys.maxint. Reverting to default value for '
+                               'max_workers')
                 return False
-
             return True
 
         # Starting Python 3.9, worker won't be putting a limit on the
         # max_workers count in the created threadpool.
         default_value = None if sys.version_info.minor == 9 \
```

Comment:
Shouldn't this be `sys.version_info.minor >= 9`?

Reply:
Currently, there isn't any 3.10, but yes, this would be updated whenever the work for 3.10 is done. As we don't know how the implementation of ThreadPoolExecutor is and what setting default value would have an impact on. Will keep that in mind.